### PR TITLE
Add pkgconfig note for macOS High Sierra

### PR DESCRIPTION
**Pull request by CorbanR**
Fix some spelling

### CODE REVIEW

File: source/guides/installing.html.md.erb [hunk 1]
```diff
@@ -19,11 +19,14 @@ To get Lucky, you need to install these first
...
-Once the required dependencies are installed, set up Lucky for your system
+Once the required dependencies are installed, set up Lucky for your system.
 
 ### On macOS
 
-Make sure you have home-brew, then run `brew install lucky`
+Make sure you have [homebrew](https://brew.sh/), then run `brew install lucky`.
+
+If you are on macOS Sierra you may need to add `export PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig`
+to your `~/.bash_profile` or `~/.zshrc`.
```
Comment (CorbanR) [inline source/guides/installing.html.md.erb]: Changed this to `~/.bash_profile` for macOS as that is the default sourced file. I'm not a zsh user, but it appears that on macOS just plain old `~/.zshrc` is more common. Having said that I would be happy to change that back to `~/.bashrc` and `~/.zshrc.local` if you prefer.

Comment (paulcsmith) [reply source/guides/installing.html.md.erb]: Those changes look pretty much perfect. The file names you used are more accurate :)

One note is that I think this is only required on macOS High Sierra. I think regular Sierra does not need it.

**Review by paulcsmith — COMMENTED** [review]
Nice changes. Thanks for catching those typos while you were at it. Just one comment about this being needs for just High Sierra

File: source/guides/installing.html.md.erb (hunk 1 above)
Comment (CorbanR) [reply source/guides/installing.html.md.erb]: Ah good point. Ill update.

**Review by paulcsmith — COMMENTED** [review]
Perfect. Thank you!

**Pull request merged**